### PR TITLE
UseDelay + rad cleanup

### PR DESCRIPTION
**Pull request by metalgearsloth**
I just wanted to fix some things from rad collector PR and ended up doing a mild rework of UseDelay to make more sense.

### CODE REVIEW

**Comment by Partmedia** [comment]
This would be easier to review if the UseDelay and rad cleanup were separate. The rad cleanup part looks fine, but I have not tested this in game. How much of the changed stuff was tested in game?

File: Content.Server/Singularity/Components/RadiationCollectorComponent.cs
Comment (AJCM-git) [inline Content.Server/Singularity/Components/RadiationCollectorComponent.cs:1]: Make these use the new default names

File: Content.Shared/Timing/UseDelayComponent.cs
Comment (AJCM-git) [inline Content.Shared/Timing/UseDelayComponent.cs:1]: same

**Review by AJCM-git — CHANGES_REQUESTED** [review]
Since you already have to merge master may as well apply new standards

**Comment by AJCM-git** [comment]
superseded by  #22502

**Pull request closed without merging**